ezyrb/gui.py

```diff
@@ -526,12 +525,8 @@ def _main(self):
 		self.root.config(menu=menubar)
 
 	def start(self):
-		"""
-		This method inizializes and starts the GUI. 
-		"""
-		self._main()
-		self.root.mainloop()
 
+		self.root.mainloop()
```

Comment:
you erased self.main() here so you have to add it at the end of the file and in the start_gui.py